inc/Print.php

```diff
@@ -725,6 +726,11 @@ public static function AdminEditUser()
 			echo '</td>
 			</tr>';
 
+			echo '<tr>
+			<td>Last detected score (dd/mm/yyyy)</td>

```

Comment:
this seems a bit unclear from an admin pov, is this "Cheats last detected at"?

Reply:
"Last detected score" sounds kinda like "last time they submitted any score"

Reply:
yep agree i kinda named without thinking